src/GUI/ccextractorGUI.c

```diff
@@ -606,16 +629,16 @@ int main(void)
 				if (main_settings.filename_count != 0)
 				{
 					nk_layout_row_static(ctx, 18, 380, 1);
-                    nk_label(ctx, concat("Input type: ", input.type[input.type_select]), NK_TEXT_LEFT);
-                    nk_label(ctx, concat("Output type: ", output.type[output.type_select]), NK_TEXT_LEFT);
-                    if(output.is_filename)
-                        nk_label(ctx, concat("Output path: ", output.filename), NK_TEXT_LEFT);
-                    else
-                        nk_label(ctx, "Output path: Default", NK_TEXT_LEFT);
-                    if(burned_subs.is_burned_subs)
-                        nk_label(ctx, "Hardsubtitles extraction: Yes", NK_TEXT_LEFT);
-                    else
-                        nk_label(ctx, "Hardsubtitles extraction: No", NK_TEXT_LEFT);
+					nk_label(ctx, concat("Input type: ", input.type[input.type_select]), NK_TEXT_LEFT);
```

Comment:
Leaks here... 
Maybe write a printf() function that works on labels - check the source code of mprintf for example.
You could do a nk_label_printf (ctx, NK_TEXT_LEFT, "Input type: %s%s", input.type[input.type_select]);
which would be a lot neater. 
That function in fact could be contributed to nuklear, since it would be quite useful for everybody.

Reply:
I didn't do anything here other than convert the mixed identation from spaces and tabs to tabs only. I'll try my best to fix this